sdk/identity/identity/README.md

```diff
@@ -143,19 +134,9 @@ const client = new KeyClient(vaultUrl, credential);
 
 ### Specifying a user assigned managed identity with the `DefaultAzureCredential`
 
-Many Azure hosts allow the assignment of a user assigned managed identity. This example demonstrates configuring the `DefaultAzureCredential` to authenticate a user assigned identity when deployed to an Azure host. It then authenticates a `KeyClient` from the [@azure/keyvault-keys](https://www.npmjs.com/package/@azure/keyvault-keys) client library with credential.
-
-```ts
-const { KeyClient } = require("@azure/keyvault-keys");
-const { DefaultAzureCredential } = require("@azure/identity");
+A relatively common scenario involves authenticating using a user assigned managed identity for an Azure resource. The `DefaultAzureCredential` makes this a relatively straightforward task that can be configured using environment variables or in-code.
```

Comment:
I generally advise against using link text like "this", "here" etc. This hinders screenreader users from being able to quickly go over the links on the page and get context. We can modify the above text as suggested below and then remove the line on "For more information..."

```suggestion
A relatively common scenario involves authenticating using a user assigned managed identity for an Azure resource. Explore the [example on user assigned managed identity with DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-a-user-assigned-managed-identity-with-defaultazurecredential) to see how this is made a relatively straightforward task that can be configured using environment variables or in-code.
```

Reply:
Thanks! I took your suggestion, but it would have applied against the wrong section so I manually applied it